src/time/time.zig

```diff
@@ -499,6 +499,10 @@ pub const Duration = struct {
         return .{ .ns = 0 };
     }
 
+    pub fn fromMinutes(m: u64) Duration {
+        return .{ .ns = m * 60 * std.time.ns_per_s };
```

Comment:
Extremely minor nit:
```suggestion
        return .{ .ns = m * std.time.ns_per_min };
```

Reply:
[145db2f](https://github.com/Syndica/sig/pull/506/commits/145db2f54bcc71b058acb2794a68ff6a699e4cf1)